telebot/__init__.py

```diff
@@ -5875,7 +5876,7 @@ def _check_filter(self, message_filter, filter_value, message):
             return False
 
     # middleware check-up method
-    def _check_middleware(self, update_type):
+    def _check_middlewares(self, update_type):
```

Comment:
I think we can rename this function to _get_middlewares

Reply:
+